packages/vitest/src/runtime/suite.ts

```diff
@@ -167,7 +167,7 @@ function createSuite() {
   suite.each = <T>(cases: ReadonlyArray<T>) => {
     return (name: string, fn: (...args: T[]) => void) => {
       cases.forEach((i, idx) => {
-        const items = toArray(i) as any
+        const items = Array.isArray(i) ? i : [i]
```

Comment:
Should `toArray()` be fixed to avoid transforming `null` into `[]` ?
Here: https://github.com/vitest-dev/vitest/blob/c23be109deb67f796bc31edb61d3acbb306fea49/packages/ui/client/composables/summary.ts#L50
Instead of `||` use `??`

Reply:
?? will still transform it into an empty array. 
we have two implementations. the one you referenced has nothing to do with the PR. I think it is useful somewhere, but I am too afraid to remove it, but here it’s it definetly not needed.

Reply:
Uh, sorry for the `??` noice. Still is strange that a `toArray` will convert `null` to `[]`, but you are right, it may be better to be explicit here.

Reply:
If a non-ternary alternative is desired, `[].concat(i)` may work.
![image](https://user-images.githubusercontent.com/24852502/173146425-f4b88d2e-b278-4822-904b-9aa0115d0c2d.png)

I'm a bit late since the PR just merged, but can still be done in a follow-up PR 😅